src/agent/debuglet.ts

```diff
@@ -106,6 +112,46 @@ const formatBreakpoints = function(
           .join('\n');
 };
 
+/**
+ * CachedPromise class is designed to support debug agent in Google Cloud
+ * Function (GCF). As GCF is a serverless environment and we wanted to make
+ * sure debug agent could always capture the snapshots. We designed this class
+ * so that GCF could wait the ListActiveBreakpoitns event completed before
```

Comment:
Nit.

ListActiveBreakpoitns -> ListActiveBreakpoints
"wait" -> "wait until"
"event completed" -> "event is completed"

Reply:
Done.